src/webview/__tests__/generateInboundEventEditSequence-test.js

```diff
@@ -437,8 +439,12 @@ describe('messages -> piece descriptors -> content HTML is stable/sensible', ()
         state: eg.reduxStatePlus({
           streams: [...eg.plusReduxState.streams, stream1, stream2],
           subscriptions: [
-            ...eg.plusReduxState.subscriptions,
             // no subscription for stream1
+            //
+            // Could spread eg.plusReduxState.subscriptions, for
+            // completeness…but then we'd invite flakes where random stream
+            // IDs in there sometimes collide with our fixed ones, changing
+            // whether a stream is subscribed
```

Comment:
More specifically: it would defeat the "no subscription for stream1", because it might happen to contain a subscription for stream1.